### PR TITLE
Add default values for most parameters

### PR DESCRIPTION
**Pull request by normen**
This PR creates default values for most parameters so they don't have to be entered in the config file for each camera. Notably it limits the frame rate to 10 fps by default as this seems to greatly improve the latency and video quality on the iOS side for my various cameras. Otherwise the values are basically left as they were. It also updates the README.

I'd be happy if the PR would not be squashed but simply merged so I can continue working in my forked repo without having messy future PRs or having to delete it each time I want to make a PR for this project :)

### CODE REVIEW

**Comment by normen** [comment]
I also added proper logging and a debugging option for ffmpeg as it leaves the cameras passwords in the log by default as it is now.

File: README.md [hunk 1]
```diff
@@ -19,21 +19,23 @@ ffmpeg plugin for [Homebridge](https://github.com/nfarina/homebridge)
           "name": "Camera Name",
           "videoConfig": {
           	"source": "-re -i rtsp://myfancy_rtsp_stream",
-            "stillImageSource": "-i http://faster_still_image_grab_url/this_is_optional.jpg",
-          	"maxStreams": 2,
-          	"maxWidth": 1280,
```
Comment (KhaosT) [inline README.md]: It doesn't make any sense to remove the width and height parameter, it's different for each camera.

File: README.md [hunk 2]
```diff
@@ -19,21 +19,23 @@ ffmpeg plugin for [Homebridge](https://github.com/nfarina/homebridge)
...
-          	"maxHeight": 720,
-          	"maxFPS": 30
```
Comment (KhaosT) [inline README.md]: Same for FPS.

**Comment by normen** [comment]
Okay, changed. I have a local change for a cappable bitrate, shall I add that too? Looking into that other -tune issue as well.

File: README.md [hunk 3]
```diff
@@ -19,21 +19,27 @@ ffmpeg plugin for [Homebridge](https://github.com/nfarina/homebridge)
           "name": "Camera Name",
           "videoConfig": {
           	"source": "-re -i rtsp://myfancy_rtsp_stream",
-            "stillImageSource": "-i http://faster_still_image_grab_url/this_is_optional.jpg",
+          	"stillImageSource": "-i http://faster_still_image_grab_url/this_is_optional.jpg"
           	"maxStreams": 2,
           	"maxWidth": 1280,
           	"maxHeight": 720,
-          	"maxFPS": 30
+          	"maxFPS": 30,
```
Comment (KhaosT) [inline README.md]: Extra `,`

**Comment by KhaosT** [comment]
Looks good :) I'll do the `-tune zerolatency` directly.

**Comment by normen** [comment]
Cool, any interest in that bitrate option?

**Comment by normen** [comment]
Heh, I would have added the parameters properly on two lines instead of one :P

Anyway, on to deleting the repo again, I'll make PRs from branches from now on, OK?

**Comment by KhaosT** [comment]
@normen What do you mean by bitrate option?

**Comment by KhaosT** [comment]
And yeah, ideally I guess creating new feature from a branch is the ideal way to go...

**Comment by normen** [comment]
Just saw that I added that in the PR already, I had it separate for a while.. There can now be a max bitrate.

**Pull request merged**